packages/npm-packages/ruby-wasm-wasi/test/unit/test_object.rb

```diff
@@ -182,6 +182,69 @@ def test_call_with_stress_gc
     GC.stress = false
   end
 
+  def test_new_embedded_object
```

Comment:
Sorry, what `embedded_object` means here?

Reply:
I was using embedded as built-in.
I have changed it to use standard built in as it is.
I think it is easier to understand now.